actions/actions.py

```diff
@@ -57,48 +149,45 @@ def backup() -> None:
     })
 
 
-def restore() -> None:
+def restore():
+    """
+    Implementation of easyrsa 'restore' action
+
+    Backup restoration process can be summarized as following:
+
+        * Selected backup is scanned and verified
+        * Contents of the backup are unpacked into <cahrm_dir>/EasyRSA/pki
+        * Data that are stored in the local database are updated
+        * All units that have relation with this easyrsa unit will be notified
+          about the certificate changes.
+    """
     pki_dir = os.path.join(easyrsa_directory, 'pki')
-    backup_name: str = function_get('name')
+    backup_name = function_get('name')
+
     if backup_name is None:
-        raise RuntimeError('Parameter \'name\' is required.')
+        raise RuntimeError("Parameter 'name' is required.")
 
     backup_path = os.path.join(PKI_BACKUP, backup_name)
 
-    shutil.rmtree(pki_dir, ignore_errors=True)
+    if not os.path.isfile(backup_path):
+        raise RuntimeError("Backup with name '{}' does not exist. Use action "
+                           "'list-backups' to list all available "
+                           "backups".format(backup_name))
+
     with tarfile.open(backup_path, 'r:gz') as tar:
```

Comment:
Suggest using `pki_tar` instead of `tar`. Having the same type of object (and usage) with the same name helps readability.